tests/twisted/roster/groups.py

```diff
@@ -154,11 +154,39 @@ def test(q, bus, conn, stream):
             EventPattern('dbus-return', method='RemoveMembers'),
             )
     assertEquals('duncan@scotland.lit', iq.stanza.query.item['jid'])
-    groups = set([str(x) for x in xpath.queryForNodes('/iq/query/item/group',
-        iq.stanza)])
+    groups = {str(x) for x in xpath.queryForNodes('/iq/query/item/group',
+        iq.stanza)}
     assertLength(1, groups)
     assertContains('Scots', groups)
 
+    # Test SetContactGroups, which didn't previously have proper coverage
```

Comment:
This part seems to leak from the `master` branch.

Reply:
Oops, sorry 😨